src/nft/components/collection/FilterButton.tsx

```diff
@@ -37,7 +37,7 @@ export const FilterButton = ({
       width={isMobile ? '44' : 'auto'}
       height="44"
       whiteSpace="nowrap"
-      color="textPrimary"
+      color="white"
```

Comment:
are we sure we want to hardcode this tow hite?

Reply:
this is a merge conflict, i changed this upstream

Reply:
![Screen Shot 2022-11-14 at 9 40 02 AM](https://user-images.githubusercontent.com/15934595/201688213-a99280f2-3901-4c6b-b387-544a180d13f6.png)

This is what it looks like with textPrimary